### DISCUSSION TITLE
Github Copilot: IntelliJ Idea stuck in Initialization

### DISCUSSION THREAD

**Discussion by jerabekfilip** [community/community#137588] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hello, I was using Github Copilot Chat fine and suddenly it is stuck in initialization. I use the latest version of idea (2024.2.1) and updated Github Copilot to the latests too. Tried to reinstall Idea, reinstall the Github Copilot plugin and nothing works. 

See the image bellow.
![image](https://github.com/user-attachments/assets/9deb3679-5452-45e0-965a-64f46bad015d)

Thanks for help!

**Comment by seanyasno** [comment]
Same about me, I'm using the same latest version of idea (2024.2.1) and I can the same issue as you have...

**Reply by panayao** [reply]
I am also having this exact same issue
IntelliJ 2024.2
CoPilot Plugin 1.5.19.6527 (also tried 1.5.20.6554)
macOS Sonoma 14.5

I found that this appears to be because CoPilot is making API requests to new servers that potentially are not whitelisted by my organization's network, or those servers are unreachable. Notably, GitHub Copilot and Chat both work in VS Code on the exact same machine.

GitHub, can you look into this for Enterprise clients as well?

**Reply by jerabekfilip** [reply]
> I am also having this exact same issue IntelliJ 2024.2 CoPilot Plugin 1.5.19.6527 (also tried 1.5.20.6554) macOS Sonoma 14.5
> 
> I found that this appears to be because CoPilot is making API requests to new servers that potentially are not whitelisted by my organization's network, or those servers are unreachable. Notably, GitHub Copilot and Chat both work in VS Code on the exact same machine.
> 
> GitHub, can you look into this for Enterprise clients as well?

I have tried disconnecting the company VPN and the issue in Intellij still persists. 

Also tried VS Code version `1.92.2` with Github copilot `v1.226.0` which does not work either. I have two github profiles, one with Copilot assigned and other with access to repos. In Intellij I am able to click `Sign in to Github`, open browser and choose different profile. But in VS Code I am getting error `[error] [auth] No access to GitHub Copilot found. You are currently logged in as XXX.` and cannot switch the profile to the one with Copilot access.

**Reply by ToFe19** [reply]
I have exactly the same problem... any solution to that?

**Comment by huyz** [comment]
Yeah still very unstable, freezing up my IntelliJ. It used to be at init time. Now when using the Chat feature. I'm avoiding the Chat view pane now.

GitHub CoPilot 1.5.23.6819
IntelliJ IDEA 2024.2.2 (Ultimate Edition), Build #IU-242.22855.74, built on September 18, 2024
macOS 14.7 23H124 (Sonoma) MacBookPro18,2 (Apple M1 Max, arm64)

**Reply by huyz** [reply]
My guess is that GitHub Copilot doesn't play nice with other AI plugins.

**Comment by wwzeng1** [comment]
If you'd like to use a better plugin, we built [Sweep AI](https://docs.sweep.dev) using native Java Swing components. It has "apply" to instantly make code changes, and we're working on next-edit capabilities in JetBrains.

I'd recommend you give it a try!

**Comment by ToFe19** [comment]
I changed “Copilot CLI” to “Local” and now everything works as expected…

<img width="397" height="352" alt="image" src="https://github.com/user-attachments/assets/d346a117-5f2e-4b50-8152-420827ff2035" />
